src/utils/api.jsx

```diff
@@ -0,0 +1,11 @@
+import axios from 'axios'
+
+const articlesApi = axios.create({
+    baseURL:"https://northcoders-news-back-end-project.onrender.com/api/articles"
+})
+
```

Comment:
Your baseURL should just be upto /api, then you can reuse it in each of the functions below with /articles or /users or /article/:article_id etc rather than having multiple instances.